gui/index.html

```diff
@@ -379,28 +379,28 @@ <h3 class="panel-title">
                     <th><span class="fa fa-fw fa-tachometer"></span>&nbsp;<span translate>CPU Utilization</span></th>
                     <td class="text-right">{{system.cpuPercent | alwaysNumber | natural:1}}%</td>
                   </tr>
-                  <tr ng-if="system.extAnnounceOK != undefined && announceServersTotal > 0">
-                    <th><span class="fa fa-fw fa-bullhorn"></span>&nbsp;<span translate>Global Discovery</span></th>
+                  <tr ng-if="system.discoveryEnabled">
```

Comment:
ALL THE DISCOVERIES

We only want to hide the line if both local and global is disabled. Otherwise it shows its status count out of the ones that are enabled, regardless of type.